cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/MultivariateAnomalyDetection.scala

```diff
@@ -300,6 +300,165 @@ class MultivariateAnomalyEstimator(override val uid: String) extends Estimator[D
   }
 }
 
+trait HasBlobHelper extends HasServiceParams {
```

Comment:
nit: we should probably rename this to something like SimpleMultiADParams as it has a bit more params than just for the blob helper